core/tests/protractor/collections.js

```diff
@@ -82,7 +110,93 @@ describe('Collections', function() {
     users.logout();
   });
 
+  it('checks for console errors in a collection with one node', function() {
+    users.login('player@collections.com');
+    browser.get('/search/find?q=');
+    browser.waitForAngular();
+    element.all(by.css(
+      '.protractor-test-collection-summary-tile-title')
+    ).filter(function (elem) {
+      return elem.getText().then(function (text) {
+        return text === 'Test Collection 2';
+      });
+    }).first().click();
+    browser.waitForAngular();
+    general.waitForSystem();
+    browser.getCurrentUrl().then(function(url) {
+      var pathname = url.split('/');
+      COLLECTION_ID = pathname[4];
+    });
+    users.logout();
+  });
+
+  it('checks for console errors in a collection with two nodes', function() {
+    users.login('player@collections.com');
+    creatorDashboardPage.get();
+    browser.waitForAngular();
+    element.all(by.css(
+      '.protractor-test-collection-card')).first().click();
+    browser.waitForAngular();
+    general.waitForSystem();
+    collectionEditor.addExistingExploration('4');
+    collectionEditor.saveDraft();
+    collectionEditor.setCommitMessage('Add Exploration');
+    collectionEditor.closeSaveModal();
+    browser.waitForAngular();
+
+    browser.get('/collection/' + COLLECTION_ID);
+    browser.waitForAngular();
+    users.logout();
+  });
+
+  it('checks for console errors in a collection with three nodes', function() {
+    users.login('player@collections.com');
+    creatorDashboardPage.get();
+    browser.waitForAngular();
+    element.all(by.css(
+      '.protractor-test-collection-card')).first().click();
+    browser.waitForAngular();
+    general.waitForSystem();
+    collectionEditor.addExistingExploration('13');
+    collectionEditor.saveDraft();
+    collectionEditor.setCommitMessage('Add Exploration');
+    collectionEditor.closeSaveModal();
+    browser.waitForAngular();
+
+    browser.get('/collection/' + COLLECTION_ID);
+    browser.waitForAngular();
+    users.logout();
+  });
+
+  it('checks for console errors in a collection with four nodes', function() {
+    users.login('player@collections.com');
+    creatorDashboardPage.get();
+    browser.waitForAngular();
+    element.all(by.css(
+      '.protractor-test-collection-card')).first().click();
+    browser.waitForAngular();
+    general.waitForSystem();
+    collectionEditor.addExistingExploration('10');
+    collectionEditor.saveDraft();
+    collectionEditor.setCommitMessage('Add Exploration');
+    collectionEditor.closeSaveModal();
+    browser.waitForAngular();
+
+    browser.get('/collection/' + COLLECTION_ID);
+    browser.waitForAngular();
+    users.logout();
+  });
+
   afterEach(function() {
-    general.checkForConsoleErrors([]);
+    browser.manage().logs().get('browser').then(function(browserLog) {
+      var i = 0, severWarnings = false, errors = 0;
```

Comment:
severWarnings (misspelled) is not used?

Also, why don't we just do the usual 

```
  afterEach(function() {
    general.checkForConsoleErrors([]);
  });
```

instead?

Reply:
In the new file I had created, I did the other one, will try with this then.

Reply:
Used previous one itself.